### DISCUSSION TITLE
✅ Can't copy output

### DISCUSSION THREAD

**Discussion by mbierman** [isontheline/pro.webssh.net#715] [Q&A] [answered]
I was hoping to use 
`xclip -sel clip` to copy things like 

`cat filename | xclip -sel clip`

to the clipboard. But I get this error
`Error: Can't open display: (null)`

[This explanation](https://stackoverflow.com/questions/18695934/error-cant-open-display-null-when-using-xclip-to-copy-ssh-public-key
) (with 56 votes) seems to address it, but I am not sure how to put this into effect with Web SSH? 

This would be so helpful! I use pbcopy on macOS all the time.

**Comment by isontheline** [comment, marked as answer]
The biggest challenge is to support X11 Forwarding. I don't know how many hours it will require.

**Reply by mbierman** [reply]
thank you, @isontheline . Is there already an open issue for this or would it help if I create one for voting/tracking purposes?

**Reply by isontheline** [reply]
In my knowledge there is no open issue related to X11 Forwarding.

> would it help if I create one for voting/tracking purposes?

Definitely yes 🙏

**Reply by mbierman** [reply]
https://github.com/isontheline/pro.webssh.net/issues/717
